qiita_pet/webserver.py

```diff
@@ -106,6 +104,7 @@ def __init__(self):
             (r"/study/sharing/", ShareStudyAJAX),
             # ORDER FOR /study/description/ SUBPAGES HERE MATTERS.
             # Same reasoning as below. /study/description/(.*) should be last.
+            (r"/study/description/sample_template/", SampleTemplateAJAX),
```

Comment:
These are not the RESTful API endpoints but qiita_pet AJAX call endpoints. These allow lazy loading the needed elements on the study info page.